examples/community/procedural-terrain-1d/README.md

```diff
@@ -8,3 +8,5 @@ Press *space* to generate a random terrain.
 Uses [Go-Perlin](https://github.com/aquilax/go-perlin).
 
 Texture by [hh316](https://hhh316.deviantart.com/art/Seamless-stone-cliff-face-mountain-texture-377076626).
+
+![](result.png)
```

Comment:
Consider adding a title for the image,. such that if it fails to load, or if someone with visual impairment visits the site, there exists an alternative text for presenting what the image is about.

E.g.

```
![Title for image](result.png)
```

Also, is result.png missing from the PR?